### PR TITLE
Adds 3 extra bags

### PR DESCRIPTION
**Pull request by WALPVRGIS**
[WIKI]

## About the PR 
This adds 3 more backpacks, including plushie bear bag and a bread bag. :'D The white variation of the studded backpack is being added but is not obtainable (including it for use in a prefab).

bp1
bp2

The Bearpack and the Bag-uette (ty to Diacorme and DisturbHerb for that name, lmao) are able to be purchased from Geoff, at 600 for the Bag-uette and 500 for the Bearpack.

## Why's this needed? 
I sprited that cursed bear backpack and at least two people wanted to see it ingame :') Also always nice to have more options for customization!

## Changelog

```changelog
(u)Walpvrgis
(+)Two new backpacks are now able to be purchased from Geoff Honkington; one is a teddy bear and one is Bread.
```

### CODE REVIEW

**Comment by nefarious6th** [comment]
him...................the bearpack

File: code/WorkInProgress/rewardsLocker.dm [hunk 1]
```diff
@@ -193,6 +193,51 @@
 			activator.set_clothing_icon_dirty()
 			M.wear_layer = MOB_BACK_LAYER_SATCHEL
 
+		else if (istype(activator.back, /obj/item/storage/backpack/studdedwhite) || activator.back.icon_state == "bp_studdedw")
```
Comment (ZeWaka) [inline code/WorkInProgress/rewardsLocker.dm:196]: so cursed

**Review by ZeWaka — APPROVED** [review]

File: code/WorkInProgress/rewardsLocker.dm (hunk 1 above)
Comment (WALPVRGIS) [reply code/WorkInProgress/rewardsLocker.dm:196]: The entire satchel reward block of code is just... UTTERLY WRETCHED

File: code/obj/item/storage/backpack_belt_etc.dm [hunk 2]
```diff
@@ -246,6 +258,30 @@
 		icon_state = "sat_itabag_black"
 		item_state = "sat_itabag_black"
 
+/obj/item/storage/backpack/satchel/studdedblack
```
Comment (Sovexe) [inline code/obj/item/storage/backpack_belt_etc.dm:261]: why is this one typed as black when the backpack version is white

**Comment by Sovexe** [comment]
how is white studded obtained?

**Comment by ZeWaka** [comment]
> how is white studded obtained?

re: the description `The white variation of the studded backpack is also being added but is not obtainable (including it for use in a prefab).`

**Review by flrsh — CHANGES_REQUESTED** [review]
I'd like to keep the claw machine for plushies only. Consider adding the new backpacks to Geoff or random civilian crates.

**Comment by WALPVRGIS** [comment]
> 
> 
> I'd like to keep the claw machine for plushies only. Consider adding the new backpacks to Geoff or random civilian crates.

No worries, just changed that up. Wasn't super attached to the claw-machine idea, it was mostly inspired by the bearpack's design. :) I liked the Geoff idea a lot, so swapped method of obtaining to that. The bags are both extremely goofy, so they fit the theme pretty well, hah.

**Review by flrsh — APPROVED** [review]

**Pull request merged**